src/middleware/log-request-errors.ts

```diff
@@ -1,4 +1,9 @@
-module.exports = (err, req, res, next) => {
-  req.log.error(err)
+import {NextFunction} from 'express'
+import {Request,Response} from './logging'
+
+module.exports = (err: Error, req: Request, res: Response, next: NextFunction) => {
+  if (req.log) {
```

Comment:
When will `req.log` not be defined?

Reply:
It should always be defined, but in order to extend the interfaces in [logging.ts](https://github.com/szeck87/probot/blob/enable-noimplicitany/src/middleware/logging.ts#L43-L51), `log` and the other properties used in the `logRequest` function have to be optional.